media_driver/agnostic/common/vp/hal/vphal.cpp

```diff
@@ -816,11 +812,7 @@ MOS_STATUS VphalState::GetStatusReport(
         }
         else
         {   // here we have the first not ready entry.
-#if LINUX
```

Comment:
Keep the Macro and change to: #if (LINUX || ANDROID)

Reply:
Sure. Thanks